src/bin/heroku.rs

```diff
@@ -31,9 +35,38 @@ use std::{env, net::IpAddr};
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
 	env_logger::init();
+
+	dotenv().expect(".env file with DATABASE_URL variable");
```

Comment:
Let's maybe make the .env file optional? So that env vars can be set via `EXPORT a=b` too, if desired by the user

Reply:
Also I believe the env_logger also reads env vars, so let's put this line above L37

Reply:
I've moved the line. However I've not made `.env` optional for now. We can consider that mechanism later as well.